docs/guide/topics/error_handling.rst

```diff
@@ -0,0 +1,361 @@
+.. currentmodule:: discord
+
+
+==========================
+Errors and Error Handling
+==========================
+
+Handling Errors
+===============
+
+When an error ocurrs in a command or an event, it will, by default, show an error in the console to denote that the error 
+ocurred and has been silently ignored. In order to avoid this, an error handler is needed.
```

Comment:
i'm not sure if its "silently ignored" if it showed an error in the console, might be bad wording here?